301-logicapps-jobscheduler/azuredeploy.parameters.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "TimerJobLogicAppName": {

```

Comment:
remove these since you have defaultValues (or will move them to variables)

Reply:
Just to be clear, then I should just have a parameters file with an empty parameters section?
"parameters": {}